packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/useQueryAutocomplete.ts

```diff
@@ -0,0 +1,21 @@
+import { useParams } from "next/navigation";
+import { SQLAutocomplete, SQLDialect } from "sql-autocomplete";
+import { Address } from "viem";
+import { useMemo } from "react";
+import { Table } from "@latticexyz/config";
+import { useChain } from "../../../../hooks/useChain";
+import { constructTableName } from "../../../../utils/constructTableName";
+
+export function useQueryAutocomplete(table?: Table) {
+  const { id: chainId } = useChain();
+  const { worldAddress } = useParams();
```

Comment:
fyi you can cast params here with
```suggestion
  const { worldAddress } = useParams<{ worldAddress: string }>();
```

maybe even
```suggestion
  const { worldAddress } = useParams<{ worldAddress: Address }>();
```
but we should prob validate that this is an address in the layout like we do for chain names

Reply:
Ah yeah, that's nice. I'm thinking for world address we should check in layout if it's a valid world. Then in the rest of the app we can safely assume that the world is valid. However, I haven't yet found a way that I like to keep params type-safe, will check it further.